neevaco/neeva-ios
# Fix ZombieTab Losing SessionData

This bug wasn't about the `SessionData` not being restored, it was never set in the first place. To fix that, when tabs were being saved and `SessionData` was created for the `SavedTab`, attach that to the regular `Tab` as well.

## Checklists
### How was this tested?

### Manual test cases
- Navigate to site and tap some links to create a history
- Switch to another tab
- Make all tabs zombies from debug settings
- Go back to the original tab and confirm BackForwardList is restored

## Issues addressed
Fixed: #3904
